### PR TITLE
Docstrings

### PR DESCRIPTION
**Pull request by Vayel**
I partly completed docstrings. I'm not sure of this PR being correct. Am I supposed to finish the job before sending a PR?

Will the commit f606be8 cause any problem?

### CODE REVIEW

**Comment by JelteF** [comment]
Looks good overall. The merge commit is indeed a problem. You should remove it, probably through this method: https://sethrobertson.github.io/GitFixUm/fixup.html#remove_deep and then rebase all your commits onto the master instead of merging it.

I will check later if all the docstrings are correct.

**Comment by Vayel** [comment]
I probably will be able to add some documentation for specific parameters but not for global ones, like `data`, `options`... for which a global view of the code appears to be necessary (or, at least, really time-saving).

BTW, I apologize for any mistake in my English.

File: pylatex/command.py [hunk 1]
```diff
@@ -23,6 +24,18 @@ class Command(BaseLaTeXClass):
...
     def __init__(self, command, arguments=None, options=None, packages=None):
+        """
+            :param command: 
+            :param arguments: 
+            :param options: 
+            :param packages: 
+            
+            :type command: str
+            :type arguments: str
+            :type options: list or :class:`parameters.Options` instance
```
Comment (JelteF) [inline pylatex/command.py]: Arguments can be a list or as well or an Arguments instance

**Comment by JelteF** [comment]
I have checked it and commented on one thing. As for the `data` parameter, you could say it is the content of that specific container.

**Comment by Vayel** [comment]
Fixed.

However, there are many possible types for a single parameter. Too many maybe? I don't know, but documenting is not a bed of roses.

I'll gradually complete the `return` and `param` fields.

**Comment by JelteF** [comment]
Thanks, looks great.

**Pull request merged**